README.md

```diff
@@ -210,6 +210,43 @@ Create a connect standalone account. Grab your development `client_id`. Put it i
 </p>
 </details>
 
+# Object Expansion
+
+Some Stripe API endpoints support returning related objects via the object expansion query parameter. To take advantage of this feature, stripity_stripe accepts
+a list of strings to be passed into `opts` under the `:expand` key indicating which objects should be expanded.
+
+For example, calling `Charge.retrieve("ch_123")` would return a charge without expanding any objects.
+
+```elixir
+%Charge{
+  id: "ch_123",
+  balance_transaction: "txn_123",
+  ...
+}
+```
+
+However if we now include an expansion on the `balance_transaction` field using
+
+```elixir
+Charge.retrieve("ch_123", expand: ["balance_transaction"])
```

Comment:
r /`expand: ["balance_transaction"]` / `[expand: ["balance_transaction"]]`

Reply:
The extra brackets here aren't required and are being removed by the formatter.

Is there an intention to use separate styling for this repo? If not I would be happy to submit a follow-up PR that applies formatting to the codebase if you would like?

Reply:
?  See your test here - https://github.com/code-corps/stripity_stripe/pull/393/files#diff-81a69829bd06582e30fdf8543ef74eb2R8

I think you need brackets around the 2nd argument -`[expand: ["balance_transaction"]]`

Reply:
In the test I am creating the keyword list separately to the function call and in this case the brackets are required. In the README example it's created inline and therefore optional (and automatically removed).

Here's the relevant docs from the Elixir style guide.

https://github.com/christopheradams/elixir_style_guide/blob/master/README.md#keyword-list-brackets

Reply:
woah.  Learn something new everyday!  🙏